src/pages/policies/[policyId]/claims/new.svelte

```diff
@@ -22,23 +23,23 @@ export let policyId = $selectedPolicyId
 
 let itemId: string
 
+$: items = $selectedPolicyItems.filter(itemCanClaimFilter) || []
```

Comment:
Based on the old code, it looks like `$selectedPolicyItems` isn't always defined (e.g. on a direct page load to this view, before the API calls have all resolved?). You may want to protect against that possibility, too:
```suggestion
$: items = $selectedPolicyItems?.filter(itemCanClaimFilter) || []
```

Reply:
It seems to handle this ok because of the await. I suppose it's possible there could be a race condition from reactive statement, so I'll check `selectedPolicyItems` directly. I don't think the ? is needed as it defaults to an empty array.